omod/src/main/resources/liquibase.xml

```diff
@@ -35,5 +35,151 @@
 			<where>property = 'metadatasharing.systemConceptSource'</where>
 		</update>
 	</changeSet>
+	
+	<changeSet id="metadatamapping-2015-10-11-1834" author="kosmik">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="metadatamapping_metadata_source" />
+			</not>
+		</preConditions>
+		<comment>Create metadata source table</comment>
+		<createTable tableName="metadatamapping_metadata_source">
+			<column name="metadata_source_id" autoIncrement="true" type="int">
+				<constraints primaryKey="true" nullable="false" />
+			</column>
+			
+			<!-- columns for OpenMRSMetadata -->
+			<column name="name" type="varchar(255)">
```

Comment:
Yes, it should be as we are using source name as an identifier. I forgot about this when I was implementing the schema as I was concentrating on OpenMRSMetadata semantics. Regarding those semantics, it actually looks like name is sometimes unique (for example in CareSetting) even though it usually is not.

Anyway, I will add a uniqueness constraint on source name.